Primo.js

```diff
@@ -265,8 +278,10 @@ function importPNX(text) {
 	}
 	
 	item.title = ZU.xpathText(doc, '//display/title');
-	if(item.title) item.title = ZU.unescapeHTML(item.title);
-	
+	if(item.title) {
+		item.title = ZU.unescapeHTML(item.title);
+	    item.title = item.title.replace(/\s*:/, ":");
```

Comment:
It's for removing a single space in front of a semicolon. In library cataloging, titles and subtitles are separate by a space-semicolon-space, but in citation styles, the space in front of the semicolon needs to be removed, and this change will prevent users from having to do it manually in Zotero